### PR TITLE
Add github_my_star table

### LINKED ISSUE #91: Add table github_my_star
**References**
Add any related links that will help us understand the resource, including vendor documentation, related GitHub issues, and Go SDK documentation.

### PR DESCRIPTION
**Pull request by VincentHardouin**
Fix #91

# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

**Comment by rajlearner17** [comment]
Hi, @VincentHardouin So nice to see this PR, thank you !! We are looking into it, should be released soon.

File: github/table_github_my_star.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	return &plugin.Table{
+		Name:        "github_my_star",
+		Description: "GitHub stars owned by you. GitHub stars are repositories.",
+		List: &plugin.ListConfig{
+			Hydrate:           tableGitHubMyStarredRepositoryList,
+			ShouldIgnoreError: isNotFoundError([]string{"404"}),
+		},
+		Columns: []*plugin.Column{
+			{Name: "starred_at", Type: proto.ColumnType_TIMESTAMP, Transform: transform.FromField("StarredAt").Transform(convertTimestamp), Description: "The timestamp when the repository was starred."},
+			{Name: "repository_full_name", Type: proto.ColumnType_STRING, Description: "The full name of the repository, including the owner and repo name.", Transform: transform.FromField("Repository.FullName")},
```
Comment (bigdatasourav) [inline github/table_github_my_star.go:24]: repository_full_name column should be first

File: docs/tables/github_my_star.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Examples
+
+### List of your starred repositories
+
+```sql
+select
+  starred_at,
+  repository_full_name
+from
+  github_my_star
```
Comment (bigdatasourav) [inline docs/tables/github_my_star.md:16]:
```suggestion
  github_my_star;
```

File: docs/tables/github_my_star.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+```sql
+select
+  gr.homepage
+from
+  github_my_star gs
+inner join
+  github_repository gr
+on
+  gs.repository_full_name = gr.full_name
```
Comment (bigdatasourav) [inline docs/tables/github_my_star.md:29]:
```suggestion
  gs.repository_full_name = gr.full_name;
```

**Comment by bigdatasourav** [comment]
@VincentHardouin please check the suggested change also please provide example query results.

**Comment by VincentHardouin** [comment]
@bigdatasourav i took into account suggestions

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Comment by bigdatasourav** [comment]
@VincentHardouin Thank you for your quick response, It will be available released soon.

**Pull request merged**